### PR TITLE
new bocks to thank you page

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/optimizing-performance-of-transcreener-fluorescence-polarization-assays-with-the-spectramax-id5-multi-mode-microplate-reader?page=thankyou
- After: https://thank-you-page--moleculardevices--hlxsites.hlx.page/en/assets/app-note/br/optimizing-performance-of-transcreener-fluorescence-polarization-assays-with-the-spectramax-id5-multi-mode-microplate-reader?page=thankyou

### CODE REVIEW

File: blocks/additional-resources/additional-resources.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	margin-top: 0;
+	padding: 0;
+}
+
+@media only screen and (min-width: 768px) {
+  .additional-resource .columns > div {
+    display: flex;
+  }
+
+  .additional-resource .columns > div > div:first-child{
```
Comment (duynguyen) [inline blocks/additional-resources/additional-resources.css:17]: maybe odd and even child selectors are better here, e.g. when you have more than 2 resources?

File: blocks/additional-resources/additional-resources.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        const fragmentElement = document.createElement('div');
+        fragmentElement.innerHTML = fragmentHtml;
+        const img = fragmentElement.querySelector('picture');
+        const imgAnchor = img.nextElementSibling;
+        if (imgAnchor) {
+          imgAnchor.innerHTML = '';
+          imgAnchor.appendChild(img);
+        }
+        const h3 = fragmentElement.querySelector('h3');
+        return { id: h3.id, title: h3.textContent, html: fragmentElement };
```
Comment (duynguyen) [inline blocks/additional-resources/additional-resources.js:33]: is it necessary to parse h3 if we don't need id and title?

File: blocks/additional-resources/additional-resources.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      }
+      return null;
+    }),
+  );
+
+  const apps = document.createElement('div');
```
Comment (duynguyen) [inline blocks/additional-resources/additional-resources.js:39]: you could use the domHelper to simplify this line and the next one.

**Comment by andreituicu** [comment]
@dev-rajneeshkumar I'm not sure I understand. I mean it's not incorrect that you created a new block, but I think you could've simply used the `Fragment` block as we discussed.
Have you tried like this?
Screenshot 2023-08-14 at 16 27 55

Did it not work?

**Comment by dev-rajneeshkumar** [comment]
@duynguyen  We have covered all your comments . Please have a look.
@andreituicu I think we are fine with current approach hence ready to merge the PR.

File: blocks/additional-resources/additional-resources.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  .additional-resource .columns > div > div:nth-child(even) img{
+    display: block;
+    margin: auto;
+  }
+
+  .additional-resource:nth-child(even) .columns > div > div:nth-child(odd) {
+		order: 2;
+    padding-right: 0;
+    padding-left: 20px;
+	}
```
Comment (andreituicu) [inline blocks/additional-resources/additional-resources.css:36]: Please fix the indendation

**Review by andreituicu — APPROVED** [review]

File: blocks/additional-resources/additional-resources.css (hunk 4 above)
Comment (dev-rajneeshkumar) [reply blocks/additional-resources/additional-resources.css:36]: It is fixed in the file 
image

**Pull request merged**